modules/ROOT/pages/_partials/registering-a-mobile-app.adoc

```diff
@@ -0,0 +1,23 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Registering a Mobile App
+
+Prerequisites:
```

Comment:
Let's make this one a header as it is in other docs `== Prerequisites`

Reply:
👍